docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+| Peer service Organization channel admin certificate<br><br>Type=admin| The admin certificate used to administer the peer and install smart contracts on the peer. Can be configured to manage the  application channel that the peer belongs to. The associated admin user is registered with the organization CA before the peer and organization MSP are created.| Depends on implementation<br><br><code>org1ca/ <br />└── org1admin <br />&nbsp;&nbsp;└── msp <br />&nbsp;&nbsp; &nbsp;&nbsp;├── admincerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── cert.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── cacerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── localhost-7053.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── keystore <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── key.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── signcerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── cert.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;└── user <br /> </code>| Transactions can continue to process successfully. <br> <br>Cannot install new smart contracts from a client application or manage the peer from the console.| https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin |
+
+### Client Certificates
+
+Client certificates are created by applications, using the Fabric SDKs or command line CA utilities. These certificates expire after one year if using the Hyperledger Fabric CA default settings.   Certificates must be re-enrolled before expiration, or the client application will not be able to connect to the Fabric nodes.
+
+Client certificates can be re-enrolled either by using command line Hyperledger Fabric CA utilities, or by using the re-enrollment method in the Fabric CA client SDK.
+
+Refer to https://hyperledger.github.io/fabric-sdk-node/release-2.2/FabricCAClient.html#reenroll__anchor.
+
+
+### Certificate Decoding
+
+Signed certificates are distinct from registrations. Registrations do not have an expiration. Signed certificates are created based on the registration and have information from the certificate authority. This information includes metadata indicating the parent certificate authority and encoded information describing the purpose of the certificate.
```

Comment:
So all instances of "signed" should be changed to "X.509" correct? Per prior comment that all certs are signed.